core/src/main/java/org/frankframework/configuration/ConfigurationUtils.java

```diff
@@ -274,7 +274,7 @@ public static boolean addConfigToDatabase(ApplicationContext applicationContext,
 
 			String activeBool = qs.getDbmsSupport().getBooleanValue(activateConfig);
 			String reloadBool = qs.getDbmsSupport().getBooleanValue(automaticReload);
-			String query = "INSERT INTO IBISCONFIG (NAME, VERSION, FILENAME, CONFIG, CRE_TYDST, RUSER, ACTIVECONFIG, AUTORELOAD) VALUES (?, ?, ?, ?, CURRENT_TIMESTAMP, ?, "+activeBool+", "+reloadBool+")";
+			String query = "INSERT INTO IBISCONFIG (NAME, VERSION, FILENAME, CONFIG, CRE_TYDST, RUSER, ACTIVECONFIG, AUTORELOAD) VALUES (?, ?, ?, ?, CURRENT_TIMESTAMP, ?, ?, ?)";
```

Comment:
@nielsm5, moet ik deze dan ook terugdraaien? Ik zou even niet weten waarom dit fout zou gaan eigenlijk.

Reply:
Gebrek aan testen... :-(

Maar ook hier weet ik niet waarom we niet `stmt.setBoolean()` kunnen gebruiken. Was daar ooit een driver-issue mee???

Reply:
Jup, daarom doen we alles als string. Sommige db's deden raar met 1/0, t/f, true/false...

Reply:
Ik draai het wel weer terug.

Reply:
> Jup, daarom doen we alles als string. Sommige db's deden raar met 1/0, t/f, true/false...

Raar. Als je `.setBoolean()` gebruikt dan zou je daar juist geen last van moeten hebben. Tenzij je column-definitie natuurlijk niet echt overeenkomt met wat die DB wil voor een boolean??